pkg/config/core/config.go

```diff
@@ -1,8 +1,27 @@
 package core
 
+import "github.com/pkg/errors"
+
 type EnvironmentType = string
 
+// mode type for multi-cluster
```

Comment:
It's more of a `// Control Plane mode`. Standalone has nothing to do with multi-cluster